lib/wrapping_integer.fz

```diff
@@ -204,3 +218,8 @@ wrapping_integers(W type : wrapping_integer W) : numerics W is
 
   # returns the number in whose bit representation all bits are ones
   all_bits_set W is abstract
+
+
+  # how many bytes does this integer use?
+  bytes i32 is
```

Comment:
This name confused me, should maybe be `byte_count`  or `byte_size`?

Also, this should be moved to `wrapping_integer.type`.